### ISSUE TITLE
renamed rule: scss/at-import-partial-extension-blacklist (deprecation)

### ISSUE THREAD

**Issue by jens-struct** [bjankord/stylelint-config-sass-guidelines#316]
Just a heads up, that there is some renaming being done in the stylelint-scss package.

```
Deprecation warnings:
 - The "scss/at-import-partial-extension-blacklist" rule is deprecated.
 - 'at-import-partial-extension-blacklist' has been deprecated, and will be removed in '7.0'. Use 'at-import-partial-extension-disallowed-list' instead. See: https://github.com/stylelint-scss/stylelint-scss/blob/v6.1.0/src/rules/at-import-partial-extension-blacklist/README.md
```

**Comment by bjankord** [comment]
Thanks for opening the issue @jens-struct and opening a PR for the fix @rmcveigh 
I plan to cut a v11.1.0 release later this evening to get this change released.

**Comment by bjankord** [comment]
This has been released in v11.1.0: https://www.npmjs.com/package/stylelint-config-sass-guidelines/v/11.1.0
